services/comments.js

```diff
@@ -4,7 +4,8 @@ const ActionModel = require('../models/action');
 const ActionsService = require('./actions');
 
 const ALLOWED_TAGS = [
-  {name: 'STAFF'}
+  'STAFF',
+  'BEST',
```

Comment:
This decision was made as we wanted it such that eventually we wanted to support tags being added dynamically, hence the structure of the array of objects instead of the array of strings.